explorer/disks

```diff
@@ -59,6 +59,13 @@ case $uname_s in
             echo 'If you can, please submit a patch.'>&2
         fi
     ;;
+    SunOS)
+        format </dev/null \
```

Comment:
uhm, it's late, but what does the `format` do? maybe add that as comment...

Reply:
It does sound scary indeed…
Though, it was the best way I could find to get Solaris to tell about the disks it knows.

Comment added.